ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed under the MIT License.
+
+namespace DataAccessLibrary
+{
+    using Microsoft.Data.Sqlite;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Windows.Storage;
+
+    /// <summary>
+    /// Data access class for the local User table in SQLite database
+    /// </summary>
+    public static class DataAccess
+    {
+        /// <summary>
+        /// Initializes the database, which is called when application starts
+        /// </summary>
+        public async static void InitializeDatabase()
+        {
+            await ApplicationData.Current.LocalFolder.CreateFileAsync("coimbra.db", CreationCollisionOption.OpenIfExists);
+            string dbpath = Path.Combine(ApplicationData.Current.LocalFolder.Path, "coimbra.db");
+            using (SqliteConnection db =
+               new SqliteConnection($"Filename={dbpath}"))
```

Comment:
`$` strings vary by the culture they're run in (i.e. your Windows language). You should wrap this with `FormattableString.Invariant` (e.g. `FormattableString.Invariant($"Filename={dbpath}")`)